src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42RttSpec.groovy

```diff
@@ -345,6 +345,69 @@ class Server42RttSpec extends HealthCheckSpecification {
         revertToOrigin([flow], flowRttFeatureStartState, initialSwitchRtt)
     }
 
+    @Tidy
+    def "Rtt statistic is available for a flow in case switch is not connected to server42"() {
+        given: "Two active switches, src has server42 connected"
+        def server42SwIds = topology.getActiveServer42Switches()*.dpId
+        def switchPair = topologyHelper.switchPairs.find {
```

Comment:
May be useful to search the reversed pairs as well

Reply:
added